.travis/stp.sh

```diff
@@ -33,6 +33,23 @@ if [ "x${STP_VERSION}" != "x" ]; then
   sudo make install
   cd ../../
 
+  # Determine STP build flags
```

Comment:
As a very very very minor thing, I would keep the options aligned:

```bash
    STP_CMAKE_FLAGS=( \
       "-DENABLE_PYTHON_INTERFACE:BOOL=OFF" \
       "-DONLY_SIMPLE:BOOL=ON" \
       "-DSTATICCOMPILE:BOOL=ON" \
     )
   else
     STP_CMAKE_FLAGS=( \
       "-DENABLE_PYTHON_INTERFACE:BOOL=OFF" \
       "-DNO_BOOST:BOOL=ON" \
       "-DBUILD_SHARED_LIBS:BOOL=OFF" \
     )
```

This should help reading the script.